src/main/java/hw6/pages/MetalAndColorsPage.java

```diff
@@ -0,0 +1,214 @@
+            case (3):
+                first = 2;
+                break;
+            case (5):
+                first = 3;
+                break;
+            default:
+                first = 4;
+                break;
+        }
+        switch (numbers[1]) {
+            case (2):
+                second = 1;
+                break;
+            case (4):
+                second = 2;
+                break;
+            case (6):
+                second = 3;
+                break;
+            default:
+                second = 4;
+                break;
+        }
+
+        oddNumbers.select(first);
+        evenNumbers.select(second);
+    }
+
+
+    //Elements checklist
+    @FindBy(css = "section[id=elements-checklist] input[type=checkbox]")
+    public Checklist earthElements;
+
+    @XPath("//input[@id='g1']")
+    public Checkbox waterCheckBox;
+
+    @XPath("//input[@id='g2']")
+    public Checkbox earthCheckBox;
+
+    @XPath("//input[@id='g3']")
+    public Checkbox windCheckBox;
+
+    @XPath("//input[@id='g4']")
+    public Checkbox fireCheckBox;
+
+    public void selectElements(String[] elements) {
+        cleanUpElements();
+        earthElements.select(elements);
+    }
+
+    public void cleanUpElements() {
+        if (waterCheckBox.isSelected()) {
+            waterCheckBox.uncheck();
+        }
+        if (earthCheckBox.isSelected()) {
+            earthCheckBox.uncheck();
+        }
+        if (windCheckBox.isSelected()) {
+            windCheckBox.uncheck();
+        }
+        if (fireCheckBox.isSelected()) {
+            fireCheckBox.uncheck();
+        }
+    }
+
+    //Vegetables dropdown with checboxes
+    @JDropdown(root = "div[ui=droplist]",
+            value = ".filter-option",
+            list = "li",
+            expand = ".caret")
+    public Dropdown vegetablesDropdown;
+
+    @XPath("//input[@id='g5']")
+    public Checkbox cucumberCheckBox;
+
+    @XPath("//input[@id='g6']")
+    public Checkbox tomatoCheckBox;
+
+    @XPath("//input[@id='g7']")
+    public Checkbox vegetableCheckBox;
+
+    @XPath("//input[@id='g8']")
+    public Checkbox onionCheckBox;
+
+
+    public void cleanUpVegetables() {
+        vegetablesDropdown.expand();
+        if (cucumberCheckBox.isSelected()) {
+            cucumberCheckBox.uncheck();
+        }
+        if (tomatoCheckBox.isSelected()) {
+            tomatoCheckBox.uncheck();
+        }
+        if (vegetableCheckBox.isSelected()) {
+            vegetableCheckBox.uncheck();
+        }
+        if (onionCheckBox.isSelected()) {
+            onionCheckBox.uncheck();
+        }
+    }
+
+    public void selectVegetable(String[] vegList) {
+        cleanUpVegetables();
+        vegetablesDropdown.select(vegList);
+    }
+
+
+    // Buttons
+    @UI("[id=submit-button]")
+    public Button submitButton;
+
+    public void clickSubmitButton() {
+        submitButton.click();
+    }
+
+    // Results section
+    @UI("[class=summ-res]")
+    public Text summaryResult;
```

Comment:
You can create only one element like WebList and get collection of strings from it